src/Sylius/Bundle/ReportBundle/Resources/public/js/prototype-handler.js

```diff
@@ -0,0 +1,61 @@
+(function ($) {
+    'use strict';
+
+    var methods = {
+        init: function(options) {
+            var settings = $.extend({
+              'prototypePrefix': false,
+              'prototypeElementPrefix': '<hr />',
+              'containerSelector': false
+            }, options);
+
+            return this.each(function() {
+                show($(this), false);
+                $(this).change(function() {
+                    show($(this), true);
+                });
+
+                function show(element, replace) {
+                    var id = element.attr('id');
+                    var selectedValue = element.val();
+                    var prototypePrefix = id;
+                    if (false != settings.prototypePrefix) {
+                        prototypePrefix = settings.prototypePrefix;
+                    }
+
+                    var prototypeElement = $('#' + prototypePrefix + '_' + selectedValue);
+                    var container;
+
+                    if (settings.containerSelector) {
+                        container = $(settings.containerSelector);
+                    } else {
+                        container = $(prototypeElement.data('container'));
+                    }
+
+                    if (!container.length) {
+                        return;
+                    }
+
+                    if (!prototypeElement.length) {
+                        container.empty();
+                        return;
+                    }
+
+                    if (replace || !container.html().trim()) {
+                        container.html(settings.prototypeElementPrefix + prototypeElement.data('prototype'));
+                    }
+                }
+            });
+        }
+    };
+
+    $.fn.handlePrototypes = function(method) {
+        if (methods[method]) {
+            return methods[method].apply(this, Array.prototype.slice.call(arguments, 1));
+        } else if (typeof method === 'object' || !method) {
+            return methods.init.apply(this, arguments);
+        } else {
+            $.error( 'Method ' +  method + ' does not exist on jQuery.handlePrototypes' );
+        }
+    };
+})(jQuery);
```

Comment:
@Zales0123 Seems like you cannot render 2 reports in one page. It uses hardcoded "canvas" id so they override each other. I tried adding report-code to the ids in js and html, but could not fix it. Could you have a look? Try rendering 2 different reports in the dashboard.